examples/chat_app/main.py

```diff
@@ -9,11 +11,35 @@
 
 
 async def update(content: ui.column) -> None:
+    # Note: Messages should come from a database
+
+    sent = False
+    # 'sent' should be determined based on the current user
+    # which requires session/user auth - outside of scope
+    # of this example
+    # For now we just alternate for every new message
+
     content.clear()
     with content:  # use the context of each client to update their ui
         for name, text in messages:
-            ui.markdown(f'**{name or "someone"}:** {text}').classes('text-lg m-2')
-        await ui.run_javascript(f'window.scrollTo(0, document.body.scrollHeight)', respond=False)
+            # A simple way to show a message:
+            # ui.markdown(f'**{name or "someone"}:** {text}').classes('text-lg m-2')
+
+            # A more advanced example is using quasar chat_message:
+            sent = not sent
```

Comment:
Perhaps. But since anyone is free to call themselves anything this would not be a great way to solve it.

I am uncertain on how to solve this in the example without making it too much of a hack - would you do this do this client side or server side?

Reply:
I'd suggest the following: `sent=name != name_input.value`.
This requires a bit refactoring to make `name_input` accessible` from the `update` function. But then it neatly reflects which `name` is sending from the current browser tab.

Reply:
But will this work when Bob is talking to (another) Bob?

Reply:
No, same names will collide. But I think that would be ok for this kind of demo.